web/src/pages/Upload.jsx

```diff
@@ -0,0 +1,143 @@
+import { UploadOutlined, CloudUploadOutlined } from "@ant-design/icons";
+import { Button, message, Upload, Card, Typography, Divider } from "antd";
+import { useState } from "react";
+
+const { Title, Text } = Typography;
+
+export const FileUploading = () => {
+	const [fileList, setFileList] = useState([]);
+	const [uploading, setUploading] = useState(false);
+
+	const handleUpload = () => {
+		const formData = new FormData();
+		fileList.forEach((file) => {
+			formData.append("file", file);
+		});
+		setUploading(true);
+
+		fetch("https://660d2bd96ddfa2943b33731c.mockapi.io/api/upload", {
+			method: "POST",
+			body: formData,
+		})
+			.then((res) => res.json())
```

Comment:
Why did you remove this error handling logic `if (!res.ok) throw new Error("Upload failed. Please try again.");` Is there a specific reason not to check non-200 responses?